script.js

```diff
@@ -1,4 +1,420 @@
+          "<br>" +
+          "Nawwww! Dealer won mannn! " +
+          "<br>" +
+          showHandTotalValues(playerHandTotalValue, compHandTotalValue);
+      }
+    } else {
+      message =
+        showPlayerAndDealerHands(playerHand, computerHand) +
+        "<br>" +
+        "Nope! There is no Blackjack. " +
+        "<br>" +
+        showHandTotalValues(playerHandTotalValue, compHandTotalValue);
+      // no blackjack - game continues >>
+      currentGameMode = gameResults;
+      console.log(message);
+    }
+
+    //  calculate sum of hand value & com
+    var playerHandTotalValue = totalHandValue(playerHand);
+    var compHandTotalValue = totalHandValue(computerHand);
+
+    //  compare total hand valuer
+    //   same value = draw
+    if (playerHandTotalValue == compHandTotalValue) {
+      message =
+        showPlayerAndDealerHands(playerHand, computerHand) +
+        "<br>" +
+        "It's a tie! Input either 'h' or 's'. " +
+        "<br>" +
+        showHandTotalValues(playerHandTotalValue, compHandTotalValue);
+      console.log(message);
+      // change game mode
+      currentGameMode = hitOrStand;
+      return message;
+    }
+
+    // player has higher value = player wins
+    else if (playerHandTotalValue > compHandTotalValue) {
+      message =
+        showPlayerAndDealerHands(playerHand, computerHand) +
+        "<br>" +
+        "Eyyyy! You got higher values! Input either 'h' or 's'." +
+        "<br>" +
+        showHandTotalValues(playerHandTotalValue, compHandTotalValue);
+      console.log(message);
+      // change game mode
+      currentGameMode = hitOrStand;
+      return message;
+    } else {
+      message =
+        showPlayerAndDealerHands(playerHand, computerHand) +
+        "<br>" +
+        "Nawwww! Dealer has higher values! Input either 'h' or 's'." +
+        "<br>" +
+        showHandTotalValues(playerHandTotalValue, compHandTotalValue);
+      console.log(message);
+      // change game mode
+      currentGameMode = hitOrStand;
+
+      return message;
+    }
+  }
+
+  // hit or stand mode
+  if (currentGameMode == hitOrStand) {
+    // player can hit
+    if (input == "h") {
+      playerHand.push(gameDeck.pop());
+      message =
+        showPlayerAndDealerHands(playerHand, computerHand) +
+        "<br> you just drew a card. <b> input either 'h' or 's'.";
+    }
+    // player can stand
+    else if (input == "s") {
+      // calculate sum of hand value & com
+      var playerHandTotalValue = totalHandValue(playerHand);
+      var compHandTotalValue = totalHandValue(computerHand);
+      console.log("p total", playerHandTotalValue);
+      console.log("c total", compHandTotalValue);
+
+      while (compHandTotalValue < 17) {
+        computerHand.push(gameDeck.pop());
+        compHandTotalValue = totalHandValue(computerHand);
+      }
+
+      //  compare total hand valuer
+      //  same value = draw
+      if (playerHandTotalValue == compHandTotalValue) {
```

Comment:
This logic seems almost identical to the one above. This is where functions come in handy. You should almost never repeat your code.